pages/common/kak.md

```diff
@@ -0,0 +1,36 @@
+# kak
+
+> Kakoune is a code editor that uses a "normal mode" to execute edition commands, and an "insert mode" to input text verbatim into files.
```

Comment:
Is there anything to shorten the description a bit? Other than that, this looks good. :thinking: :+1: